src/scala/org/pantsbuild/zinc/compiler/OutputUtils.scala

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2012 Typesafe, Inc. <http://www.typesafe.com>
+ */
+
+package org.pantsbuild.zinc.compiler
+
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{FileVisitResult, Files, Path, Paths, SimpleFileVisitor}
+import java.util.jar.{JarEntry, JarOutputStream}
+import scala.collection.mutable
+import sbt.util.Logger
+
+object OutputUtils {
+
+  /**
+   * Jar the contents of output classes (settings.classesDirectory) and copy to settings.outputJar
+   */
+  def createClassesJar(settings: Settings, log: Logger) = {
```

Comment:
I'd recommend creating one test that sanity checks this method: to make that easier, I'd recommend replacing `Settings` with explicit arguments for the classesDirectory and outputJar (and then maybe make the timestamp optional).

An example unit test for this code is over here: https://github.com/pantsbuild/pants/tree/master/tests/scala/org/pantsbuild/zinc/analysis